pkg/binutils/latest.go

```diff
@@ -20,7 +21,16 @@ import (
 func GetLatestReleaseVersion(releaseURL string) (string, error) {
 	// TODO: Question if there is a less error prone (= simpler) way to install latest avalanchego
 	// Maybe the binary package manager should also allow the actual avalanchego binary for download
-	resp, err := http.Get(releaseURL)
+	request, err := http.NewRequest("GET", releaseURL, nil)
+	if err != nil {
+		return "", fmt.Errorf("failed to create request for latest version from %s: %w", releaseURL, err)
+	}
+	token := os.Getenv(constants.GithubAPITokenEnvVarName)
```

Comment:
1. I guess you already thought of this, but is there no way to keep this out of the code and only in the CI context?
2. On the other hand, could rate limiting be applied to normal operations, affecting cli users?

Reply:
Regarding 2, no issue for an user. Rate limiting applies currently only to subnet create operations, the user may need to create 60 subnets within an hour to reach so limit, which is improbable.
Regarding 1, env var seems a good compromise.